test/powershell/Modules/Microsoft.PowerShell.Utility/Group-Object.Tests.ps1

```diff
@@ -130,6 +130,10 @@ Describe "Group-Object" -Tags "CI" {
         $result[0].Name | Should -Be ""
         $result[0].Group | Should -Be '@{X=}'
     }
+
+    It "Should not throw with format-like strings containing curly braces (issue #20711)" {
+        { '{', '}', '{0}' | Group-Object } | Should -Not -Throw
```

Comment:
Please create positive test (check exact result) and remove issue number from title.

Reply:
@iSazonov , hope it's okay now (test failures are unrelated).